src/pages/Donate/Donate.jsx

```diff
@@ -43,7 +77,8 @@ export default function Donate() {
 				   onChange={handleInputChange}
 				   value={donateAmount}
 			/>
-			<button className={"donateButton"} onClick={handleDonateButtonClick}>{intl.formatMessage({id: "donate"})}</button>
+			{/* <button className={"donateButton"} onClick={handleDonateButtonClick}>{intl.formatMessage({id: "donate"})}</button> */}
+			<div id="paypal-donate-button-container" style={{ marginTop: "20px"}}></div>
```

Comment:
Is there a reason we are creating this element using document.createElement()? Would it be cleaner to create a standalone component for this with the useEffect so that it does not trigger unnecessarily?